recipes/vigra/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set version = "1.11.0" %}
+
+package:
+  name: vigra
+  version: {{ version }}
+
+source:
+  {% set version_tag = "Version-%s" % version.replace(".", "-") %}
+  fn: vigra-{{ version_tag }}-src.tar.gz
+  url: https://github.com/ukoethe/vigra/archive/{{ version_tag }}.tar.gz
+  sha1: 67788d181e759e625769bf0678887718d3dc02e2
+
+build:
+  skip: true  # [win]
+  number: 0
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - gcc         # [linux]
```

Comment:
Need a compiler with C++11 support. So, we are installing it from the `defaults` channel, as well. Eventually, it will be possible to drop this if/when the new docker container comes into play.